packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -101,11 +101,11 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
       );
 
       //emit configuration.json
-      const namespace = options.namespace ?? root.Name;
+      const namespace = root.Name;
       const configurations: Configuration = {
         "output-folder": ".",
         namespace: namespace,
```

Comment:
should we just remove this from the configuration.json?
autorest.csharp could add this back if it still needs it.